src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx

```diff
@@ -32,16 +33,17 @@ export default class ClinicalInformationPatientTable extends React.Component<
         attribute: string;
         value: string;
     }): string {
-        let ret: string;
-        switch (data.attribute) {
-            case 'Overall Survival (Months)':
-                ret = parseInt(data.value, 10).toFixed(0);
-                break;
-            default:
-                ret = data.value;
-                break;
+        if (isNaN(data.value as any)) {
+            return data.value;
+        } else {
+            const parsedFloat = parseFloat(data.value);
+            if (parsedFloat != parseInt(data.value)) {
```

Comment:
Should we use strict inequality instead?
```suggestion
            if (parsedFloat !== parseInt(data.value)) {
```

Reply:
Good point! Adjusted for this.